boomer/boomer.go

```diff
@@ -127,6 +131,9 @@ func (b *Boomer) runWorker(n int) {
 		TLSHandshakeTimeout: time.Duration(b.Timeout) * time.Millisecond,
 		Proxy:               http.ProxyURL(b.ProxyAddr),
 	}
+	if b.H2 {
+		http2.ConfigureTransport(tr)
```

Comment:
@rakyll if `http2.ConfigureTransport(tr)` fails, it stick with the origin transport (HTTP/1 in this case), is it really needed to handle the return error?